backend/typescript/middlewares/validators/camperValidators.ts

```diff
@@ -38,3 +38,41 @@ export const createCamperDtoValidator = async (
 
   return next();
 };
+
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable-next-line import/prefer-default-export */
+export const updateCamperDtoValidator = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  if (req.body.camp && !validatePrimitive(req.body.camp, "string")) {
+    return res.status(400).send(getApiValidationError("camp", "string"));
+  }
+  if (
+    req.body.formResponses &&
+    !validateFormResponses(req.body.formResponses, "string", "string")
+  ) {
+    return res
+      .status(400)
+      .send(getApiValidationError("formResponses", "mixed", true));
+  }
+  if (
+    req.body.dropOffType &&
+    !validatePrimitive(req.body.dropOffType, "string")
+  ) {
+    return res.status(400).send(getApiValidationError("dropOffType", "string"));
+  }
+  if (req.body.registrationDate && !validateDate(req.body.registrationDate)) {
+    return res
+      .status(400)
+      .send(getApiValidationError("registrationDate", "Date string"));
+  }
```

Comment:
can we actually remove this? we shouldn't be able to update registrationDate either